instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-jersey-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jaxrsclient/v2_0/JerseyClientInstrumentationModule.java

```diff
@@ -35,6 +36,13 @@ public JerseyClientInstrumentationModule() {
     super("jaxrs-client", "jaxrs-client-2.0", "jersey-client", "jersey-client-2.0");
   }
 
+  @Override
+  public String[] additionalHelperClassNames() {
+    return new String[] {
+      "org.glassfish.jersey.client.ResponseCallbackWrapper",
+    };
+  }
```

Comment:
Can you use `isHelperClass()` instead? I think we'll get rid of `additionalHelperClassNames` soon, since `isHelperClass()` does the same thing but better.

```suggestion
  @Override
  public boolean isHelperClass(String className) {
    return className.equals("org.glassfish.jersey.client.ResponseCallbackWrapper");
  }
```